test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -43,23 +43,23 @@ def test_failed_purchase
     assert_equal '2003', response.error_code
   end
 
-  def test_successful_authorize_and_capture
-    omit 'Not yet implemented'
-
+  def test_successful_authorize#_and_capture
```

Comment:
Happy to handle this in some less goofy way :D didn't want to lose info about the standard being to test these together.